### PR TITLE
AG-10574

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-10574

### CODE REVIEW

**Comment by jacobp100** [comment]
This introduces new styling primitives throughout the series. All series are using them for tooltips, and a few are using them throughout (i.e. updateNodes). It will be a future task to refactor this to remove duplication

https://ag-grid.atlassian.net/browse/AG-13538

File: packages/ag-charts-enterprise/src/series/map-line/mapLineSeries.ts [hunk 1]
```diff
@@ -280,9 +280,9 @@ export class MapLineSeries extends TopologySeries<
             const sizeValue = sizeValues?.[datumIndex];
             const labelValue = labelValues?.[datumIndex];
 
-            const color: string | undefined =
-                colorScaleValid && colorValue != null ? colorScale.convert(colorValue) : undefined;
-            const size = sizeValue != null ? sizeScale.convert(sizeValue, true) : undefined;
+            // const color: string | undefined =
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/map-line/mapLineSeries.ts]: ?

Comment (jacobp100) [reply packages/ag-charts-enterprise/src/series/map-line/mapLineSeries.ts]: Updated

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**